sensors/depth_cam.urdf.xacro

```diff
@@ -80,7 +80,7 @@
         </camera>
         <plugin name="${name}_controller" filename="libgazebo_ros_openni_kinect.so">
           <baseline>0.2</baseline>
-          <alwaysOn>true</alwaysOn>
+          <alwaysOn>true</alwaysOn>roslaunch navigation navigation.launch map:=aws_hospital planner:=dwa
```

Comment:
I guess the roslaunch command got accidantely copied in here. please remove.

Reply:
OK